UnityProject/Assets/Scripts/Items/Food/Donut.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System.Collections;
+using UnityEngine;
+
+
+public class Donut : Edible
+{
+    [Tooltip("The amount of brute damage healed for a player who has a secuirty role.")]
+    public int SecurityHealBruteDamage = 5;
+    private int securityHealBruteDamage => SecurityHealBruteDamage;
+
+
+    /// <summary>
+	/// Checks if the player eating the donut has a secuirty job and heals them if they are part of security.
+    /// If the player is not part of secuirty then nothing special will be triggered.
+	/// </summary>
+	public override void Eat(PlayerScript eater, PlayerScript feeder)
+	{
+		if (CheckForJob(eater))
+		{
+            Heal(eater);
+		    Eat(eater, feeder, NutritionLevel);
+		}
+		else
+		{
+		    Eat(eater, feeder, NutritionLevel);
+		}
+	}
+
+    /// <summary>
+	/// Heals all body parts that have brute damage.
+	/// </summary>
+    private void Heal(PlayerScript player)
+    {
+        var livingHealth = player.GetComponent<LivingHealthBehaviour>();
+        foreach (BodyPartBehaviour BodyPart in livingHealth.BodyParts)
+		{
```

Comment:
here indentation looks weird. Are you using spaces instead of tabs?

Reply:
In vscode it looks normal to me, and I use tabs yeah.
Working on fixing a lot of issues here now.